onnxruntime/test/onnx/TestResultStat.h

```diff
@@ -61,19 +61,19 @@ class TestResultStat {
     }
 
     for(const auto& s:result.failed_kernels) {
-        AddNotImplementedKernels(s);
+        AddFailedKernels(s);
     }
 
-    for(const auto& s:result.failed_test_cases) {
-        AddNotImplementedKernels(s);
+    for(const auto& p:result.failed_test_cases) {
+        AddFailedTest(p);
     }
 
     return *this;
   }
 
  private:
-  onnxruntime::OrtMutex m_;
+  mutable onnxruntime::OrtMutex m_;
   std::unordered_set<std::string> not_implemented_kernels;
   std::unordered_set<std::string> failed_kernels;
-  std::unordered_set<std::string> failed_test_cases;
+  std::vector< std::pair<std::string, std::string> > failed_test_cases;
```

Comment:
>std::pair<std::string, std::strin [](start = 15, length = 33)

Need a comment that describes the content of the pair

Reply:
done